src/runtime/auto-prompt-manager.js

```diff
@@ -272,7 +272,12 @@ export class AutoPromptManager {
       (clientConfig?.autoPromptConfig?.clientDisplayTrigger
         ?.displayDelaySeconds || 0) * SECOND_IN_MILLIS;
 
-    if (shouldShowAutoPrompt) {
+    const interventionDisplayedIsAutoPromptType =
+      this.interventionDisplayed_ === null ||
+      // Only need to check contributions because Subsciptions interventions
+      // will not be saved.
+      this.isContribution_({autoPromptType: this.interventionDisplayed_.type});
```

Comment:
@kristenwang adding this check here due to edge case: https://github.com/subscriptions-project/swg-js/blob/main/src/runtime/auto-prompt-manager.js#L579

Do we plan to support this long term, or is this only maintained while we transition contributions to audience actions?